agent/utils/ticker_test.go

```diff
@@ -38,9 +38,8 @@ func TestTickerHappyCase(t *testing.T) {
 			break
 		}
 	}
-
-	if times < 10 || times > 100 {
-		t.Error("Should tick at least 10 but less than 100 times: ", times)
+	if times < 8 || times > 100 {
```

Comment:
I am also curious: was 8 the smallest number that showed up before?

Reply:
Yes exactly, We don't know for sure that this is the right number to have in the test case but this is the smallest number that had showed up.

Reply:
Can you add a comment here to give a little bit explanation about why we choose the number?